src/storage/mapping/ExtensionBasedMapper.ts

```diff
@@ -1,16 +1,24 @@
 import { promises as fsPromises } from 'fs';
 import * as mime from 'mime-types';
 import type { ResourceIdentifier } from '../../ldp/representation/ResourceIdentifier';
-import { TEXT_TURTLE } from '../../util/ContentTypes';
+import { APPLICATION_TRIG, TEXT_TURTLE } from '../../util/ContentTypes';
 import { NotImplementedHttpError } from '../../util/errors/NotImplementedHttpError';
 import { joinFilePath, getExtension } from '../../util/PathUtil';
 import { BaseFileIdentifierMapper } from './BaseFileIdentifierMapper';
 import type { FileIdentifierMapperFactory, ResourceLink } from './FileIdentifierMapper';
 
 export class ExtensionBasedMapper extends BaseFileIdentifierMapper {
+  public static readonly mimeFallback: Record<string, string> = {
+    'application/trig': 'trig',
+  };
```

Comment:
I would dynamically generate this object based on the input overrideTypes (and just pick the first one or so for duplicates), makes it easier should another type be needed soon.